### PR TITLE
Switched to use .material on the body class instead of further down the stack

### PR DESCRIPTION
**Pull request by chrismou**
Switched to prefix elements with `.gmusic-theme.material` to take advantage of the body class, rather than relying on the presence of a `.material` selector further down the stack - which, as noted in #15, I don't seem to have anymore, causing the now playing list text to render in black against the dark grey background.  (A/B testing? Update not yet propagated worldwide?)

Aside from how I've done it here, another option would be to simply leave out `.material` from the selectors completely (which tbh might be OK, if you're suffixing them all with `!important` anyway). Let me know what your thoughts are.

I mentioned in #15 - I have no idea why `npm run build` is creating so many changes in `dist/gmusic-theme.js` (compared to other commits). That's the only command I'm running after making the changes.

### CODE REVIEW

**Comment by MarshallOfSound** [comment]
I'll take a peek in a sec, but if I had to guess at the diff.  I'd say the color library I use got updated.  So you have a newer version and it got browserified and therefore changed the dist :+1:

File: lib/_constants.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  CLASS_NAMESPACE: 'gmusic-theme',
+  CLASS_NAMESPACE: 'gmusic-theme.material',
```
Comment (MarshallOfSound) [inline lib/_constants.js]: The question we should be asking is, do we even need `.material` in the namespace.  As far as I know / can tell we can kill it completely.

Comment (chrismou) [inline lib/_constants.js]: Yeah, I mentioned that in the PR notes. Seeing as we're suffixing them all with !important anyway, it's probably not required.

I think I'm going to just strip it out.  If nothing else, it'll look a lot tidier.

Comment (MarshallOfSound) [inline lib/_constants.js]: Sounds good.

And lol, I read the PR notes.  Came back 30 minutes later and read the diff.  Guess I should do it all at once next time :+1:

**Comment by chrismou** [comment]
Removed the hard coded `.material` reference

**Comment by MarshallOfSound** [comment]
LGTM :+1:

**Comment by chrismou** [comment]
How are you managing releases on this repo?  Shall I tag a new patch version?

**Comment by MarshallOfSound** [comment]
I'll handle it, I don't think you have perms on NPM yet (I shall remedy that) :+1:

**Pull request merged**